notify/impl.go

```diff
@@ -562,7 +477,7 @@ func (n *OpsGenie) Notify(ctx context.Context, as ...*types.Alert) error {
 		if err != nil {
 			return
 		}
-		s, err = n.tmpl.ExecuteTextString(name, data)
+		s, err = n.tmpl.ExecuteTextString(name, template.NewData(groupLabels(ctx), as...))
```

Comment:
This is recalculating the template data each time, it'd be better to do it just once